NDTensors/test/test_dense.jl

```diff
@@ -38,6 +38,11 @@ NDTensors.dim(i::MyInd) = i.dim
 
       Aview = A[2:3, 2:3]
       @test dims(Aview) == (2, 2)
+      ## Added for issue 1431 create a tensor from 
+      ## a sliced view of another tensor
+      Acopy = Tensor(NDTensors.storage(Aview), (1,4))
```

Comment:
@mtfishman I am not sure if I should add a test here since this constructor uses an internal `NDTensors` function.

Reply:
Is the internal function you are referring to `NDTensors.storage(...)`? I think it is ok to use that in the tests.

Reply:
Yes it is. Thanks!